assets/html/templates/site-header.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * AMPConf static site-header template.
+ *
+ * @package AMPConf
+ */
+
+?>
+
+<header class="module module-header">
```

Comment:
@kopepasah So in the `site-header` and `site-footer` are we going with a module pattern or the BEM naming convention?

If we want to go with the BEM, I'd suggest replacing the `.module` and `.container` with the `.wrap` layout component so that we reduce code duplication.

Reply:
@delawski sure, that sounds fine.